frontend/src/api/HttpCommon.js

```diff
@@ -5,7 +5,7 @@ import axios from 'axios'
  */
 
 const Service = axios.create({
-  baseURL: '10.0.3.187/solr',
+  baseURL: '/service',
```

Comment:
Great. Now, in `Search.js` you still have the IP hardcoded, making your proxy useless. in your proxy you've mapped `http://10.0.3.187/solr/` to `/service`, so now you can change the get calls to something like ```Service.get(`/employees/select?q=${key}:${value}`)```.